src/Tools/FirstRunCertGenerator/test/CertificateManagerTests.cs

```diff
@@ -164,7 +164,7 @@ public void EnsureCreateHttpsCertificate_ReturnsExpiredCertificateIfVersionIsInc
             now = new DateTimeOffset(now.Year, now.Month, now.Day, now.Hour, now.Minute, now.Second, 0, now.Offset);
             _manager.EnsureAspNetCoreHttpsDevelopmentCertificate(now, now.AddYears(1), path: null, trust: false, subject: TestCertificateSubject);
 
-            CertificateManager.AspNetHttpsCertificateVersion = 2;
+            CertificateManager.AspNetHttpsCertificateVersion = 3;
```

Comment:
```suggestion
            CertificateManager.AspNetHttpsCertificateVersion = CertificateManager.AspNetHttpsCertificateVersion + 1;
```

Reply:
Sounds like a good change, but I'm not going to re-trigger the build for this, as it's not a significant improvement.

Reply:
Everytime we update the version, we are going to need to update this test. People are going to forget and have to respin CI each time. Why not fix the test now s.t. we don't hit this issue in the future.

Reply:
We don't do this often, we can change it next time.